include/libtorrent/storage.hpp

```diff
@@ -695,7 +695,9 @@ namespace libtorrent
 	// what to do when it's actually touching the file
 	struct fileop
 	{
-		virtual int file_op(int file_index, boost::int64_t file_offset, int size
+		virtual int file_op(int const file_index
```

Comment:
I would prefer to change the .cpp file. the fact that the function doesn't change its incoming arguments should not leak out into its interface